usecases/auth/authorization/types.go

```diff
@@ -35,7 +35,7 @@ const (
 
 var Actions = map[string]string{
 	CRUD:   "manage",
-	CRU:    "view",
+	CRU:    "manage",
```

Comment:
**Thought:** now that this is being changed I wonder: should `CRU` be `"edit"` since we have that the built-in role of `editor` has `CRU` on everything?

Reply:
i agree the problem we don't have yet that type of permissions, so it shall be `edit permissions`  so 3 actions 
`create_`, `update_`, `read_` but i think that will need different way to handle it given that type is built in